### PR TITLE
CSS at-rules: rewrite see also and description list

### PR DESCRIPTION
**Pull request by estelle**
Make the at-rules page more useful:

- update definitions of each at-rule
- add link to defining module
- add missing at-rules
- rewrite see-also to make it specific to the page
- update counter-style page see also section (moving the demo to examples area)

Note: 
- the `@document` is intentionally omitted as the page is 2+years deprecated and is to be removed.
- the `@view-transitions` does not have a module landing page, yet.

### CODE REVIEW

File: files/en-us/web/css/@counter-style/index.md [hunk 1]
```diff
@@ -121,11 +121,11 @@ The above code produces the following result:
 
 {{EmbedLiveSample('Specifying symbols with counter-style', '', '300')}}
 
-See more examples on the [demo page](https://mdn.github.io/css-examples/counter-style-demo/).
+See more examples on the [demo page](https://mdn.github.io/css-examples/counter-style-demo/) ([code](https://github.com/mdn/css-examples/tree/main/counter-style-demo)).
 
 ### Ready-made counter styles
 
-Find a collection of over 100 `counter-style` code snippets in the [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/) document. This document provides counter styles that meet the needs of languages and cultures around the world.
+Find a collection of over 100 `counter-style` code snippets in the [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/) document. This document provides counter-styles that meet the needs of languages and cultures around the world.
```
Comment (bsmth) [inline files/en-us/web/css/@counter-style/index.md:128]: Are we using "counter styles" or "counter-styles" here? The original looks okay here 🤔

File: files/en-us/web/css/at-rule/index.md [hunk 2]
```diff
@@ -41,32 +39,36 @@ Statement at-rules end in a semicolon. There are several statement at-rules, des
...
+  - : A conditional group rule that applies its content if the container meets the [`<container-condition>`](/en-US/docs/Web/CSS/@container#container-condition)s ([CSS containment](/en-US/docs/Web/CSS/CSS_containment)).
 - {{cssxref("@font-face")}}
-  - : Describes the aspect of an external font to be downloaded.
+  - : Defines font resource locations, both local and external, along with the style characteristics for when those resources are used with a declared {{cssxref("font-family")}} ([CSS fonts](/en-US/docs/Web/CSS/CSS_fonts)).
 - {{cssxref("@font-feature-values")}} (plus `@swash`, `@ornaments`, `@annotation`, `@stylistic`, `@styleset` and `@character-variant`)
-  - : Define common names in {{cssxref("font-variant-alternates")}} for feature activated differently in OpenType.
-- {{cssxref("@keyframes")}}
-  - : Describes the aspect of intermediate steps in a CSS animation sequence.
+  - : Control font display per font-family by defining font-specific alternates, or custom names, to feature indexes in {{cssxref("font-variant-alternates")}} in OpenType ([CSS fonts](/en-US/docs/Web/CSS/CSS_fonts)).
```
Comment (bsmth) [inline files/en-us/web/css/at-rule/index.md:51]:
```suggestion
  - : Controls font display per font-family by defining font-specific alternates, or custom names, to feature indexes in {{cssxref("font-variant-alternates")}} in OpenType ([CSS fonts](/en-US/docs/Web/CSS/CSS_fonts)).
```

File: files/en-us/web/css/at-rule/index.md [hunk 3]
```diff
@@ -41,32 +39,36 @@ Statement at-rules end in a semicolon. There are several statement at-rules, des
...
 - {{cssxref("@scope")}}
-  - : A conditional group rule that will apply its content if the device meets the criteria of the given condition.
+  - : Defines a scope in which to apply them to selected elements and the styles to apply to the elements in that scope ([CSS cascade and inheritance](/en-US/docs/Web/CSS/CSS_cascade)).
 - {{cssxref("@starting-style")}}
-  - : A conditional group rule that will apply starting transition if the browser meets the criteria of the given condition.
+  - : Define the starting property values for an element to transition from when the element receives its first style update, such as when transitioning from `display: none` ([CSS transitions](/en-US/docs/Web/CSS/CSS_transitions)).
 - {{cssxref("@supports")}}
-  - : A conditional group rule that will apply its content if the browser meets the criteria of the given condition.
+  - : A conditional group rule applies its content if the browser supports the CSS features of the given condition ([CSS conditional rules](/en-US/docs/Web/CSS/CSS_conditional_rules)).
+- {{cssxref("@view-transitions")}}
```
Comment (bsmth) [inline files/en-us/web/css/at-rule/index.md:70]: typo?

```suggestion
- {{cssxref("@view-transition")}}
```

**Comment by bsmth** [comment]
Thanks, Estelle. A couple of comments for you to have a look at 👀

File: files/en-us/web/css/@counter-style/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/css/@counter-style/index.md:128]:
```suggestion
Find a collection of over 100 `counter-style` code snippets in the [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/) document. This document provides counters that meet the needs of languages and cultures around the world.
```

**Review by bsmth — APPROVED** [review]
LGTM, thank you

**Pull request merged**